src/measure.rs

```diff
@@ -79,8 +106,19 @@ async fn setup_subscriber(
     Ok(subscriber)
 }
 
-struct Provider {
-    pub provider_interface: Box<dyn ProviderInterface>,
+fn create_databroker_endpoint(host: String, port: u64) -> Result<Endpoint> {
+    let databroker_address = format!("{}:{}", host, port);
+
+    let endpoint = tonic::transport::Channel::from_shared(databroker_address.clone())
+        .with_context(|| "Failed to parse server url")?;
+    let endpoint = endpoint
+        .initial_stream_window_size(1000 * 3 * 128 * 1024) // 20 MB stream window size
```

Comment:
Comments mismatch with actual values (actual 393MB, but comment says 20 MB)
Imho, this should be changed.

It's very unrealistic to have so much memory on a target embedded device.

Reply:
WIll comment out and fixed in the future.